src/commands/distribute/groups/create.ts

```diff
@@ -81,16 +81,16 @@ export default class CreateDistributionGroupCommand extends AppCommand {
 
   private async createDistributionGroup(client: AppCenterClient, app: DefaultApp) {
     try {
-      const createDistributionGroupRequestResponse = await out.progress(
+      await out.progress(
         "Creating distribution group...",
-        clientRequest((cb) => client.distributionGroups.create(app.ownerName, app.appName, this.distributionGroup, cb))
+        client.distributionGroups.create(app.ownerName, app.appName, this.distributionGroup)
       );
-      if (createDistributionGroupRequestResponse.response.statusCode >= 400) {
-        throw createDistributionGroupRequestResponse.response.statusCode;
-      }
     } catch (error) {
       if (error === 409) {
         throw failure(ErrorCodes.InvalidParameter, `distribution group ${this.distributionGroup} already exists`);
+      } else if (error >= 400) {
+        //TODO:
```

Comment:
Can you elaborate please?

Reply:
Forgot to remove it. This `else` is not needed.